frontend/src/pages/TermPlanner/ExportPlannerMenu/ExportPlannerMenu.tsx

```diff
@@ -8,8 +10,26 @@ type Props = {
 };
 
 const ExportPlannerMenu = ({ plannerRef }: Props) => {
-  const exportFormats = ['png', 'jpg'];
+  const exportFormats = ['png', 'jpg', 'json'];
   const exportFields = { fileName: 'Term Planner' };
+  const planner = useSelector((state: RootState) => state.planner);
+
+  const jsonFormat = {
+    startYear: planner.startYear,
+    numYears: planner.numYears,
+    isSummerEnabled: planner.isSummerEnabled,
+    years: planner.years,
+  };
```

Comment:
This doesn't really matter at all (for now) but, it might be good to add some form of version numbering here too, just in case the structure ever changes.

Reply:
The main thing I can for-see is if sometimes in the future this also has an option to export marks.

Reply:
Yup, this^ Please save the redux version!